### DISCUSSION TITLE
backup certificates

### DISCUSSION THREAD

**Discussion by ale82x** [beeyev/Mikrotik-RouterOS-automatic-backup-and-update#10] [Ideas]
Hello,
thank you for script.

i'm testing it, and works great for backup my mikrotik device.

but i now if i "backup" my device
/system backup save .......
i can't restore it to another device, correct?

so if i have a hap ac3 with some user configured in openvpn (server certificates and client certificates)
i backup it with your script, and hap ac3 crash. i change it with an another hap ac3 and restore rsc correct?
but how i restore openvpn certs?

or can i implement script with backup certs on mikrotik?

thank you

**Comment by TerAnYu** [comment]
Hello, @ale82x !
The current version of the script does not yet save certificates.
I can recommend my solution for storing all certificates and configurations on ftp: https://github.com/TerAnYu/mikrotik_ftp_backup

**Comment by ale82x** [comment]
ok, thank you
i can try use it, or edit for use SFTP protocol.

thank you
